### ISSUE TITLE
Unable to delete clients assigned to Org

### ISSUE THREAD

**Issue by TjebbeVQ** [Velocidex/velociraptor#2060]
Deleting clients inside an org is looking for the wrong file.

<html>
<body>

Timestamp | Level | message
2022-09-09T09:41:23Z | DEFAULT | Running query on behalf of user TVQ
2022-09-09T09:41:23Z | DEFAULT | <green>Starting</> query execution.
2022-09-09T09:41:23Z | DEFAULT | Deleting client C.21271f16b27e1196
2022-09-09T09:41:23Z | DEFAULT | client_delete: While opening /clients/C.21271f16b27e1196.db: open /opt/velociraptor/orgs/OT5B0/clients/C.21271f16b27e1196.db: no such file or directory: file does not exist
2022-09-09T09:41:23Z | DEFAULT | Query Server.Utils.DeleteClient: Emitted 0 rows
2022-09-09T09:41:23Z | DEBUG | Query Stats: {"RowsScanned":1,"PluginsCalled":3,"FunctionsCalled":3,"ProtocolSearch":0,"ScopeCopy":7}

</body>
</html>

The backend file seems to be called: C.21271f16b27e1196-OT5B0.db (with the ORG ID included)

Any way to manually delete the client from command line / backend system?

**Comment by scudette** [comment]
Thanks for reporting this! It was a bug in the way the `Server.Utils.DeleteClient` artifact over sanitized the client id.

I submitted a fix but you can just delete clients in the notebook like this:

```sql
SELECT client_delete(client_id="XXXX", really_do_it=TRUE)
FROM scope()
```

Where client id is the full string including the org designation e.g. `C.0c9026bb25c56633-OVBP0`

**Comment by scudette** [comment]
Fixed by #2062
